endpoints/publicProxy/google.go

```diff
@@ -145,8 +145,7 @@ func configureGoogleOauth(cfg *OauthConfig, tls bool) error {
 		} else {
 			authCheckInterval = i
 		}
-
-		SetZrokCookie(w, cfg.CookieDomain, rDat.Email, tokens.AccessToken, "google", authCheckInterval, key)
+		SetZrokCookie(w, cfg.CookieDomain, rDat.Email, tokens.AccessToken, "google", authCheckInterval, key, token.Claims.(*IntermediateJWT).Host)
```

Comment:
What is the difference between `token` parsed from `state` and `tokens.AccessToken`?

Reply:
`tokens.AccessToken` is used to authenticate requests to Google APIs (and GitHub in counterpart implementation `github.go`), while the `token` parsed from `state` is used to maintain state and pass information between the request and callback.

Here's [a zrok public share running this code](https://radial.ken.demo.openziti.org/) if you want to see it in action.